tests/integration/infrastructure/scalingo-api_test.js

```diff
@@ -162,5 +162,164 @@ describe('scalingo-api', () => {
         });
       });
     });
+    describe('#topNQueries', () => {
+      describe('when there is more active queries than topN', () => {
```

Comment:
when there is more active  than N active queries

Reply:
Corrigé